packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -131,6 +131,23 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
     this._groupedList && this._groupedList.scrollToIndex(index, measureItem);
   }
 
+  public focusIndex(
```

Comment:
I see a public API being added, but no change to a .types file.

It would also be great to use enzyme and write some tests to cover these things.

Reply:
Done.